### PR TITLE
qiskit: 0.5.7 -> 0.6.1

### PR DESCRIPTION
**Pull request by pandaman64**
###### Motivation for this change
update 3 packages

###### Things done

- [x] Tested using sandboxing ([nix.useSandbox](http://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](http://nixos.org/nix/manual/#sec-conf-file) on non-NixOS)
- Built on platform(s)
   - [x] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [x] Tested compilation of all pkgs that depend on this change using `nix-shell -p nox --run "nox-review wip"`
- [x] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

---

### CODE REVIEW

File: pkgs/development/python-modules/qiskit/default.nix [hunk 1]
```diff
@@ -2,6 +2,8 @@
 , isPy3k
 , buildPythonPackage
 , fetchPypi
+, fetchurl
+, python
```
Comment (xeji) [inline pkgs/development/python-modules/qiskit/default.nix]: `fetchurl` and `python` are not used anywhere. Why add them?

File: pkgs/development/python-modules/qasm2image/default.nix [hunk 2]
```diff
@@ -12,13 +12,13 @@
 
 buildPythonPackage rec {
   pname = "qasm2image";
-  version = "0.7.0";
+  version = "0.8.0";
 
   src = fetchFromGitHub {
-    owner = "nelimeee";
+    owner = "pandaman64";
```
Comment (xeji) [inline pkgs/development/python-modules/qasm2image/default.nix]: What's the reason for switching to a fork? It seems to be behind the original repo.

**Comment by pandaman64** [comment]
fixed contaminations.

**Comment by xeji** [comment]
@GrahamcOfBorg build python36Packages.qiskit python36Packages.qasm2image python36Packages.IBMQuantumExperience python27Packages.IBMQuantumExperience

**Pull request merged**